### PR TITLE
New version for openCARP packages, v12.0

### PR DESCRIPTION
**Pull request by MarieHouillon**
Add a new version for the packages of the openCARP ecosystem, opencarp, meshtool and py-carputils.

Also sets the default cmake build type to "Release".

### CODE REVIEW

File: var/spack/repos/builtin/packages/meshtool/package.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Meshtool(MakefilePackage):
 
     version("master", branch="master", preferred=True)
     # Version to use with openCARP releases
+    version("oc12.0", commit="867431d")
```
Comment (haampie) [inline var/spack/repos/builtin/packages/meshtool/package.py:19]: same commit as oc11.0, typo or intentional?

**Review by haampie — COMMENTED** [review]
See above

File: var/spack/repos/builtin/packages/meshtool/package.py (hunk 1 above)
Comment (MarieHouillon) [reply var/spack/repos/builtin/packages/meshtool/package.py:19]: This is intentional

**Review by tldahlgren — COMMENTED** [review]
Confirmed all of the commits exist.

**Review by alalazo — APPROVED** [review]

**Pull request merged**